lib/core/enhanceError.js

```diff
@@ -17,11 +17,11 @@ module.exports = function enhanceError(error, config, code, request, response) {
   }
 
   error.request = request;
-  error.response = response;
+  error.response=response;
   error.isAxiosError = true;
 
-  error.toJSON = function() {
-    return {
+  error.toJSON = function( ) {
```

Comment:
format: style mismatch:
```suggestion
  error.toJSON=function() {
```

`457c2d3c` ␣ at column 15 should be removed.
`457c2d3c` ␣ at column 17 should be removed.
`457c2d3c` ␣ at column 27 should be removed.